pytorch-notebook/environment.yml

```diff
@@ -6,9 +6,9 @@ channels:
  - nodefaults
 dependencies:
  - cudatoolkit=11
- # needed for pytorch to work
+ # needed for pytorch to work ?
  # https://github.com/NVIDIA/MinkowskiEngine/issues/354#issuecomment-880555195
- - anaconda::libopenblas
+ # - anaconda::libopenblas
```

Comment:
conda-lock seems to hang now and not handle this mixing of channels anymore, so disabling to unblock other environments... added a timeout in CI to skip and still push successful lockfiles in the future